app-web/src/templates/resourceType.js

```diff
@@ -60,16 +56,16 @@ export const ResourceType = ({
     query = '';
   }
 
-  results = useSearch(query, index);
+  results = useSearch(query);
   // this is defined by ?q='' or ?q=''&q=''..etc
   // if query is empty we prevent the search results empty from being rendered
   // in addition the topics container is prevented from not rendering because
   // the query is present
   const queryIsEmpty = isQueryEmpty(query);
   const resourceTypeConst = RESOURCE_TYPES[pageContext.resourceTypeConst];
-  const nodes = flattenGatsbyGraphQL(allDevhubSiphon.edges).concat(
-    flattenGatsbyGraphQL(allGithubRaw.edges),
-  );
+  const nodes = flattenGatsbyGraphQL(allDevhubSiphon.edges)
+    .concat(flattenGatsbyGraphQL(allGithubRaw.edges))
```

Comment:
this is incorrect. All Github Raw nodes are associated with markdown nodes. I think the algolia index needs to be reconfiguired.